core/domain/summary_services.py

```diff
@@ -230,13 +266,40 @@ def get_exploration_metadata_dicts(exploration_ids, editor_user_id=None):
 
 def get_displayable_exp_summary_dicts_matching_ids(
         exploration_ids, editor_user_id=None):
-    """Given a list of exploration ids, optionally filters the list for
+    """Get a summary of explorations in human readable form from
+    exploration ids.
+
+    Given a list of exploration ids, optionally filters the list for
     explorations that are currently non-private and not deleted, and returns a
     list of dicts of the corresponding exploration summaries. This function can
     also filter based on a user ID who has edit access to the corresponding
     exploration, where the editor ID is for private explorations. Please use
     this function when needing summary information to display on exploration
     summary tiles in the frontend.
+
+    Args:
+        exploration_ids: list(str). List of exploration id
+        editor_user_id: str of None. User id to filter exploration. If None the
```

Comment:
of --> or

str or None. If provided, the returned value ...[insert description here]... . Otherwise, the returned list is not filtered.